### ISSUE TITLE
Component is not compatible with NVDA

### ISSUE THREAD

**Issue by johannesh2** [vaadin/vaadin-cookie-consent#39]
Results from #12 

NVDA on Win 7 (both with IE 11 and Chrome)
2.1. when the page loads: NVDA does not announce the cookie consent popup (neither in Chrome nor in IE11)
2.2. when the focus is moved to the first focusable element in the popup (the 'learn more' link): NVDA reads the content of the dialog twice
Chrome:
Cookie consent dialog. This website uses cookies to ensure you get the best experience. Learn more about cookies. This website uses cookies to ensure you get the best experience. Button "Learn more about cookies".
IE11:
Dialog "This website uses cookies to ensure you get the best experience. Learn more." This website uses cookies to ensure you get the best experience. Button "Learn more about cookies".

**Comment by johannesh2** [comment]
Spike timebox 1d
